### ISSUE TITLE
Cannot connect with ED25519 key

### ISSUE THREAD

**Issue by KSSea** [mimecuvalo/firessh#43]
I don't know if this is only for ED25519 keys, but I am unable to connect to a server which I know accepts connections from other clients using the key.  It fails with "Unable to make a connection.  Please try again."

**Comment by mimecuvalo** [comment]
Indeed, ED25519 keys are not currently supported.  Patches welcome!
